frontend/services/entities/hosts.ts

```diff
@@ -50,6 +50,74 @@ export default {
       },
     });
   },
+  exportHosts: (options: ILoadHostsOptions | undefined) => {
```

Comment:
Very similar call as loadHosts(), just removed the pagination parameters and had it route to HOSTS_REPORT instead of HOSTS